webpack.config.js

```diff
@@ -7,19 +9,25 @@ module.exports = {
     publicPath: `/`
   },
   module: {
-    loaders: [
+    rules: [
       {
         test: /.jsx?$/,
-        loader: `babel-loader`,
         exclude: /node_modules/,
-        query: {
-          presets: [`es2015`, `react`]
-        }
-      },
-      {
-        test: /\.json$/,
-        loader: `json`
+        use: [
+          {
+            loader: `babel-loader`,
+            options: {
+              presets: [`es2015`, `react`]
+            }
+          }
+        ]
       }
     ]
-  }
+  },
+  plugins: [
```

Comment:
Please uplift first, then add in more functionality to the config as follow-up. This is good to have but **only** for a production build. Source maps are actually super bloaty so it's far better to know which environment we're in and then either minify (for prod) without source maps, or not minify at all (for dev).